__tests__/lib/units/ph.test.ts

```diff
@@ -0,0 +1,8 @@
+import { PH } from "../../../src/lib/units/ph";
+
+it("should only accept values between 0 <= value <= 14", () => {
+  expect(() => new PH(0)).not.toThrow();
+  expect(() => new PH(14)).not.toThrow();
+  expect(() => new PH(-0.1)).toThrow();
+  expect(() => new PH(14.1)).toThrow();
+});
```

Comment:
Add one test between 0 and 14 not to throw?

Reply:
see https://github.com/sensor-network/open-data-portal/pull/65/commits/7f306adb1f8a9cedc1c8538460440e10a88698b0